test/conformance/kuttl/generate/clusterpolicy/standard/clone/sync/cpol-clone-list-sync-create/01-assert.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: sync-secret-with-multi-clone
+status:
+  ready: true
```

Comment:
We're moving away from status specified like this and towards an object. Check what @eddycharly has done with some recent modifications.

Reply:
done